src/main/java/com/softserve/academy/spaced/repetition/service/DeckService.java

```diff
@@ -5,46 +5,185 @@
 import com.softserve.academy.spaced.repetition.utils.exceptions.NotAuthorisedUserException;
 import com.softserve.academy.spaced.repetition.utils.exceptions.NotOwnerOperationException;
 import org.springframework.data.domain.Page;
+
 import java.util.List;
 
+/**
+ * This interface proceeds all operations with decks.
+ */
 public interface DeckService {
+    /**
+     * Adds a new deck to the category
+     *
+     * @param deck       the deck which will be added to the category.
+     * @param categoryId category`s id for which the deck will be added.
+     */
     void addDeckToCategory(Deck deck, Long categoryId);
 
+    /**
+     * Adds a new deck to the course
+     *
+     * @param deck       the deck which will be added to the category.
+     * @param categoryId id of the category for which belongs the course.
+     * @param courseId   course`s id to which the deck will be added.
```

Comment:
to which -> by which

Reply:
it`s the course id to which  we add the deck